test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -151,8 +155,24 @@ if [ "$FLOW_AGGREGATOR" == "true" ]; then
     if [[ $FLOW_COLLECTOR != "" ]]; then
         echo "Generating manifest with all features enabled along with FlowExporter feature"
         $THIS_DIR/../../../../hack/generate-manifest.sh --mode dev --all-features > "${ANTREA_YML}"
-
-        $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $FLOW_COLLECTOR > "${FLOW_AGG_YML}"
+        if [[ $FLOW_COLLECTOR == "ELK" ]]; then
+            echo "Deploy ELK flow collector"
+            echo "Copying ELK flow collector folder"
+            pids=()
+            scp -F ssh-config -r $THIS_DIR/../../../../build/yamls/elk-flow-collector k8s-node-control-plane:~/ &
+            pids+=($!)
+            waitForNodes "${pids[@]}"
```

Comment:
this doesn't seem necessary, you are scp'ing to a single machine, so you can do it in the foreground

Reply:
Yes there are no other background processes.